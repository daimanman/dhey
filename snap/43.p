{"C":1,"Z":"","N":1,"Remark":"压力测试","Url":"http://localhost:9188/go1","Method":"POST","TaskId":43,"Type":"FIXED","FileId":"4","Status":0,"Err":"","QH":{},"P":{"info":"TEsss颠三倒四ST","name":"DXwwM"},"Payload":"","StartTime":"2018-08-20T11:54:25.0377434+08:00","EndTime":"0001-01-01T00:00:00Z","Duration":0}