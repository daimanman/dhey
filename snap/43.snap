{"AvgTotal":0.0471237,"Fastest":0.0471237,"Slowest":0.0471237,"Average":0.0471237,"Rps":21.220744551043317,"AvgConn":0.0070071,"AvgDNS":0.0070071,"AvgReq":0,"AvgRes":0.0010013,"AvgDelay":0.0381007,"ConnMax":0.0070071,"ConnMin":0.0070071,"DnsMax":0.0070071,"DnsMin":0.0070071,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0010013,"DelayMax":0.0381007,"DelayMin":0.0381007,"Lats":[0.0471237],"ConnLats":[0.0070071],"DnsLats":[0.0070071],"ReqLats":[0],"ResLats":[0.0010013],"DelayLats":[0.0381007],"Total":47123700,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":45,"SizeReq":45,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0471237,"Count":1,"Frequentcy":1},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0},{"Mark":0.0471237,"Count":0,"Frequentcy":0}],"TaskId":43,"Duration":0.0471237}